load_tests/load_test.py

```diff
@@ -17,6 +18,20 @@
 else:
     THROUGHPUT_LIST = json.loads(os.environ['THROUGHPUT_LIST'])
 
+# Input Logger Data
+INPUT_LOGGERS = [
+    {
+        "name": "stdstream",
+        "logger_image": "075490442118.dkr.ecr.us-west-2.amazonaws.com/load-test-fluent-bit-app-image:latest",
+        "fluent_config_file_path": "./load_tests/logger/stdout_logger/fluent.conf"
```

Comment:
This file appears to actually be empty... and it looks like we just use the task def to set the output options in all these tests? Why is a config file needed?

Reply:
Oh I see, this is because for TCP tests you need config file for the TCP input, but not for the other tests? Would be nicer if you could just put NULL here, but not a big deal.

Reply:
To keep the control flow similar for the two input loggers and any other future loggers, let's not change this.